prow/config.yaml

```diff
@@ -82,7 +82,7 @@ blocked_branches: &blocked_branches
       teams:
       - release-managers-1-2
   release-1.3: &release13
-    <<: *blocking_merge
+    protect: true
```

Comment:
Isn't that already done below?

Reply:
it was part of the `blocking_merge` block.. yaml magic